src/boot/stylesheets/main.scss

```diff
@@ -690,12 +690,41 @@ body {
     				display: block;
     			}
     		}
+
+			code {
+				font: $code;
+				font-size: 90%;
+				color: $code-color;
+				background-color: $code-background;
+				border: 1px solid $code-border;
+				border-radius: 3px;
+				padding-left: 2px;
+				padding-right: 2px;
+			}
     	}
 
     	.wpnc__paragraph:first-of-type {
     		margin-top: 0;
     	}
 
+		.wpnc__body-list {
+			list-style: disc;
+			padding: 0 16px;
+		}
+
+		.wpnc__body-todo {
+			list-style: none;
+			padding: 0 16px;
+
+			.wpnc__todo-done:before {
+				content: '✓ '
+			}
+
+			.wpnc__todo-not-done:before {
+				content: '✗ '
+			}
+		}
+
     	blockquote {
     		margin: 0 $padding-medium;
```

Comment:
Right now blockquotes look weird because they do not have a bottom margin. I think before the last commit, they were wrapped in a `p` maybe.

Change margin to this and it should look better:

`margin: 0 $padding-medium $padding-medium;`